openpdf/src/main/java/com/lowagie/text/Document.java

```diff
@@ -109,8 +112,24 @@
     /**
      * @since    2.1.6
      */
-    private static final String RELEASE = VersionBean.VERSION.getImplementationVersion();
-    private static final String OPENPDF_VERSION = OPENPDF + " " + RELEASE;
+    private static final String RELEASE;
+    private static final String OPENPDF_VERSION;
+
+    static {
+        String releaseVersion = "";
+        try (InputStream input = Document.class.getClassLoader().getResourceAsStream("com/lowagie/text/version.properties")) {
+            Properties prop = new Properties();
+            // load a properties file
+            if (input != null) {
+                prop.load(input);
+                releaseVersion = prop.getProperty("bundleVersion", "");
```

Comment:
`releaseVersion` variable is redundant, value can be assigned directly to `RELEASE`.